app/views/invitations/index.html.erb

```diff
@@ -1,11 +1,11 @@
 <!--Descriptive page name, messages and instructions-->
```

Comment:
I think [/views/invitations/_invitation.html.erb](https://github.com/otwcode/otwarchive/blob/master/app/views/invitations/_invitation.html.erb) might also need to be updated -- I got `Required parameter missing: invitation` when I tried to use the email field on my dev environment.